test/src/module/common/cryptoTest.c

```diff
@@ -329,9 +329,42 @@ testRun(void)
         TEST_RESULT_VOID(ioFilterFree(hash), "    free hash");
 
         // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("md5 hash - zero bytes");
+
         TEST_ASSIGN(hash, cryptoHashNew(strNew(HASH_TYPE_MD5)), "create md5 hash");
         TEST_RESULT_STR_Z(varStr(ioFilterResult(hash)), HASH_TYPE_MD5_ZERO, "    check empty hash");
 
+        // Exercise most of the conditions in the local MD5 code
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("md5 hash - mixed bytes");
+
+        TEST_ASSIGN(hash, cryptoHashNew(strNew(HASH_TYPE_MD5)), "create md5 hash");
+        TEST_RESULT_VOID(ioFilterProcessIn(hash, BUFSTRZ("1")), "    add 1");
```

Comment:
From lines 332 to 366, we're using TEST_TITLE so why are some of the TEST_RESULT_* comment parameters including leading spaces? Since these appear to be new tests, we should treat the formatting of the output the way we now format tests.